docs/platform-support.rst

```diff
@@ -215,3 +215,33 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - Yes
      - 
 
+
+=============================================
+Supported Precompiled Images for DOCA Drivers
+=============================================
+
+--------
+Overview
+--------
+
+To save startup (compilation) time and operational effort, precompiled DOCA driver images are available for common OS/flavor/kernel/architecture variants.
```

Comment:
What is a flavor?

Reply:
Kernel flavors: NVIDIA, AWS, Azure, and Oracle...